### ISSUE TITLE
Some Target Combos & Geneijin not auto-blocked. Supers not blocked when combod & More.

### ISSUE THREAD

**Issue by Shodokan** [Grouflon/3rd_training_lua#9]
After twins frame data were added the dummy still does not block the mp > hp > b+hp target combo or anything in geneijin whatsoever.

Can still get around this with recording feature obviously.

I'm happy so many of the other things were added though. This is going to be an amazing tool for the community.

To add to the other guy's "links don't work" thing (tested with dudley, necro, ken, chun, gouki/akuma, ryu).  Hes right.  They also "on block" will force hits.  So if you do f+mk link with dudley vs a force blocked opponent they will get hit by the super after blocking the f+mk .  If you do any normal into a super the super will not be blocked with most characters (tested with chun, dudley, makoto, akuma/gouki, ryu, necro), .  However characters like ken if you do Target combo into super it will get blocked til the last few hits.  Remy's sa2 is blocked but sa1 is not.  Necro blocked super still hitsThere are a lot of issues here.

**Comment by Grouflon** [comment]
Yeah, the blocking/parrying feature works on prerecorded frame data and is kind of a not 100% reliable hand crafted method unfortunately.
I think it's good enough for simple stuff like testing frame data and wether a move is safe or not, but I think the amount of work necessary to make it fool proof is very very high.
Imo, putting effort in making great recording tools that allow to recreate the exact setups we want instead would be a better choice.
But for Yun's target combo, i didn't knew that this target combo existed, so it's likely that it has just never been recorded :) I'll fix that for the next version.
